README.md

```diff
@@ -1 +1,82 @@
-# HELLO_MICROVERSE-2023
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Table of Contents
+
+  * About the Project
+   - Built With
+
+  *Getting Started
+    - Prerequisites
+    - Setup
+    - Install
+    - Usage
+    - Run test
+    - Deployment
+
+  * Authors
+
+  * Contributing
+
+  * Show your support
+
+  * Acknowledgements
+
+  * License
```

Comment:
- [ ] **[OPTIONAL]** Kindly consider to format corretly the table. You can do that by just copy pasting the table from the [README Template](https://github.com/microverseinc/readme-template) and savinge the file (the table should automatically update). This will help users navigate through your documentation easily 😺